Makefile

```diff
@@ -422,7 +422,7 @@ endif
 CONTRIB_BINARY_DIR := contrib/bin
 CLI_BINARY_NAME := broker-cli
 
-BUILD_CMD := go build -o $(CONTRIB_BINARY_DIR)/$(CLI_BINARY_NAME) ./cmd/cli
+BUILD_CMD := go build -o $(CONTRIB_BINARY_DIR)/$(CLI_BINARY_NAME) ./contrib/cmd/cli
```

Comment:
Ah... I see what happened here. iirc, there used to be a separate `Makefile` under `contrib`. When they were consolidated, it would appear this path (and those below) weren't adjusted accordingly.

It's gone unnoticed because we rely more often on an e2e test suite these days than the CLI and when I use the CLI (I think I am one of the few contributors who does), I tend to ad-hoc compile with `go run contrib/cmd/cli/*.go ...`

This is a good catch. Thank you for fixing!